x/chainlink/module.go

```diff
@@ -84,7 +86,10 @@ func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Rout
 
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the module.
 func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
-	// TODO: enable this using protoc-gen-grpc-gateway
+	err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+	if err != nil {
+		panic(err) // FIXME
```

Comment:
`RegisterGRPCGatewayRoutes` will be called upon the module gets initiated, so I guess it's fine to panic here, what you think?

Reply:
Think is fine to panic here too, and I can't see other alternative to handle the error

Reply:
cool, I saw you added a comment there saying `FIXME`, that's why I asked, but it is correct.

Reply:
yeah i think panic is fine here too